crates/sui-indexer/diesel.toml

```diff
@@ -4,8 +4,11 @@
 [print_schema]
 file = "src/schema.rs"
 patch_file = "src/schema.patch"
+#file = "src/schema_v2.rs"
+#patch_file = "src/schema_v2.patch"
```

Comment:
uncomment?

Reply:
I was worried this would get mixed up with v1. But if we don't think there will be changes to v1, I would uncomment the v2 rows